components/public-api/gitpod/v1/user.proto

```diff
@@ -11,13 +11,69 @@ option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
 service UserService {
   // GetAuthenticatedUser allows to retrieve the current user.
   rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {}
+
+  // UpdateUser updates the properties of a user.
+  rpc UpdateUser(UpdateUserRequest) returns (UpdateUserResponse) {}
+
+  // SetWorkspaceAutoStartOptions updates the auto start options for the Gitpod Dashboard.
+  // +internal - only used by the Gitpod Dashboard.
+  rpc SetWorkspaceAutoStartOptions(SetWorkspaceAutoStartOptionsRequest) returns (SetWorkspaceAutoStartOptionsResponse) {}
 }
 
 message GetAuthenticatedUserRequest {}
 message GetAuthenticatedUserResponse {
   User user = 1;
 }
 
+message SetWorkspaceAutoStartOptionsRequest {
+  string user_id = 1;
+  repeated WorkspaceAutostartOption workspace_autostart_options = 2;
+  message WorkspaceAutostartOption {
+    string clone_url = 1;
+    string organization_id = 2;
+    optional string workspace_class = 3;
+    optional EditorReference editor_settings = 4;
+    optional string region = 5;
+  }
+}
+message SetWorkspaceAutoStartOptionsResponse {}
+
+message UpdateUserRequest {
+  string user_id = 1;
+  optional string name = 2;
+  optional string accepted_privacy_policy_date = 3;
+  optional ProfileDetails profile = 4;
+  message ProfileDetails {
+    optional string last_updated_details_nudge = 1;
+    optional string accepted_privacy_policy_date = 2;
+    optional string company_name = 3;
+    optional string email_address = 4;
+    optional string job_role = 5;
+    optional string job_role_other = 6;
+    repeated string exploration_reasons = 7;
+    repeated string signup_goals = 8;
+    optional string signup_goals_other = 10;
+    optional string onboarded_timestamp = 11;
+    optional string company_size = 12;
+  }
+  optional string email_address = 5;
+  optional EditorReference editor_settings = 6;
+  optional EmailNotificationSettings email_notification_settings = 7;
+  message EmailNotificationSettings {
+    optional bool allows_changelog_mail = 1;
+    optional bool allows_devx_mail = 2;
+    optional bool allows_onboarding_mail = 3;
+  }
+  optional WorkspaceTimeoutSettings workspace_timeout_settings = 8;
+  message WorkspaceTimeoutSettings {
+    optional google.protobuf.Duration inactivity = 1;
+    optional bool disabled_disconnected = 2;
```

Comment:
we have another workspace timeout in `Refind workspace proto PR` https://github.com/gitpod-io/gitpod/pull/19138/files#diff-393d8178f08b8814eea780cf779ac93bc55fd0b90588e907a825cbb51de9f33aR250

Should them be aligned?

Reply:
Let's align them in implementation PR. 
TBH, this is already confusing enough. Changed that already twice, and it's unclear how to map them. In the beginning we'll need to map them in order to maintain compatibility in FE shim, right? 

Here I did the bare minimum for the conversion. 
https://github.com/gitpod-io/gitpod/pull/19142/files#diff-8fdb520274403fe903bd0ad9c9094f3951b4940b9cf838aa5c36b6548f9aafbdR964-R967

Happy to align, if there is an example for the conversion available.